Formula/tarantool.rb

```diff
@@ -14,8 +14,11 @@ class Tarantool < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "autoconf"
```

Comment:
Are `autoconf`, `automake` and `libtool` necessary? And at run-time (not only build)? This seems unlikely, being a cmake-based project.

Reply:
Right, we really need these packages, due to we need the newer version of curl, the reason of it described here: 
https://github.com/tarantool/tarantool/issues/4318

Reply:
OK

Reply:
These should still be build dependencies no? Especially if you're statically linking.

Reply:
Ok, I've removed the changes for the versions of Tarantool and left only real fixes: added dependencies and changed 2.1 branch to 2.2 which was mistakenly left unchanged at the last pushed update.